src/tasks/migrations/run.js

```diff
@@ -0,0 +1,91 @@
+const chalk = require('chalk')
+const { isEmpty } = require('lodash')
+
+const {
+  getPathToFile,
+  checkFileExists,
+  processMigration,
+  getStoriesByComponent,
+  getNameOfMigrationFile
+} = require('./utils')
+
+/**
+ * @method runMigration
+ * @param  {Object} api       API instance
+ * @param  {String} component component name
+ * @param  {String} field     field name
+ * @param  {{ isDryrun?: boolean, migrationPath?: string }} options disable execution
+ * @return {Promise<{ executed: boolean, motive?: string }>}
+ */
+const runMigration = async (api, component, field, options = {}) => {
+  const migrationPath = options.migrationPath || null
+  try {
+    const fileName = getNameOfMigrationFile(component, field)
+    const pathToFile = getPathToFile(fileName, migrationPath)
+    const fileExists = await checkFileExists(pathToFile)
+
+    if (!fileExists) {
+      throw new Error(`The migration to combination ${fileName} doesn't exists`)
+    }
+
+    console.log(
+      `${chalk.blue('-')} Getting the user defined migration function`
+    )
+    const migrationFn = require(pathToFile)
+
+    if (typeof migrationFn !== 'function') {
+      throw new Error("The migration file doesn't export a function")
+    }
+
+    console.log(
+      `${chalk.blue('-')} Getting stories for ${component} component`
+    )
+    const stories = await getStoriesByComponent(api, component)
+
+    if (isEmpty(stories)) {
+      console.log(`${chalk.blue('-')} There are no stories for component ${component}!`)
+      return Promise.resolve({
+        executed: false,
+        motive: 'NO_STORIES'
+      })
+    }
+
+    for (const story of stories) {
+      try {
+        console.log(
+          `${chalk.blue('-')} Processing story #${story.name}`
```

Comment:
I would take story.full_slug as story.name is not unique and the user will not find it